### DISCUSSION TITLE
How to display multiple POI types at once?

### DISCUSSION THREAD

**Discussion by StacktraceException** [osmandapp/OsmAnd#16216] [Maps Related]
Hello, when I'm cycling and exploring the area, then often it would be very useful to display multiple POI and place types;
such as for example: libraries, specific brand places (like for example maybe show me all Patagonia stores nearby too) which I may like to visit, and wherever else I may need to visit that day.  

This reminds me of a filter or something, by the way, such as searching for only those bicycle shops which have a self repair area. (Each POI can have its own specific tags assigned).

Stopping constantly and checking manually makes it all a lot less enjoyable. 

Surely, many people would benefit from such functionality. Unfortunately, I do not find any way to achieve this. Can anything be done towards such an important improvement?

Kind regards  :).

EDIT: to clarify: I'm aware of this https://osmand.net/docs/user/map/point-layers-on-map/   but it does not show how to only display a specific search query list of POIs, such as showing only Patagonia stores for example and not all the clothing stores. Maybe a solution: Can custom categories be added there maybe?

**Comment by dmpr0** [comment]
Hi!
You can create custom categories with applied filters, save them, and enable needed in Menu – Configure Map – POI Overlay – Multi-selection mode.

**Comment by mikehgentry** [comment]
Search -> Categories -> Custom search

lets you choose things very selectively (if you want Hotels + Bicycle parking visible, you can do that). Once you've saved a custom search you can edit it through the same menu (it's a bit hidden - hit the switches icon then the three dots), and you can assign it to a Quick Action too.
